packages/editor/src/components/rich-text/index.native.js

```diff
@@ -356,6 +356,7 @@ export class RichText extends Component {
 					}
 					text={ { text: html, eventCount: this.lastEventCount } }
 					onChange={ this.onChange }
+					onFocus={ this.props.onFocus }
```

Comment:
I noticed the `onFocus` props is set here in `RichText` implementation but not in `PlainText` - it just works due to this https://github.com/WordPress/gutenberg/blob/master/packages/editor/src/components/plain-text/index.native.js#L34 what do you think about making it explicit there? Like, adding a 
`onFocus={ this.props. onFocus }`  for `TextInput` within `/plain-text/index.native.js`, so it is clear that the listener has to be set (and is set).